Dates.Recurring.Tests/WeeklyRecurrenceTests.cs

```diff
@@ -54,5 +55,30 @@ public void Weekly_EveryThirdWeek()
             Assert.Equal(new DateTime(2015, 2, 10), weekly.Next(new DateTime(2015, 1, 27)));
             Assert.Equal(new DateTime(2015, 2, 13), weekly.Next(new DateTime(2015, 2, 10)));
         }
+
+        [Fact]
+        public void Weekly_EveryWeek_TwoDaysAfterDateTime()
+        {
+            // Arrange.
+            DateTime startDate = new DateTime(2015, 1, 1);
+
+            IRecurring weekly = Recurs
+                .Starting(startDate)
+                .Every(1)
+                .Weeks()
+                .OnDay(startDate.AddDays(2).DayOfWeek)
```

Comment:
Can you please change this to `DayOfWeek.Saturday`, so the test reads slightly cleaner?